repairDuplicateVersions.js

```diff
@@ -0,0 +1,447 @@
+/* eslint-disable max-len */
```

Comment:
This file doesn't have any tests, not sure the standard for testing in this project

Reply:
It's a valid point, originally s3utils started as a small set of simple scripts, now that it's becoming bigger and used for more and more things the question of automated tests arise. For the detection script `verifyBucketSproxydKeys` I added a few unit tests for specific helper functions that are complex enough to benefit from them, here I did not but probably a couple functions could be tested as well.

We don't have the support needed for integration or end-to-end tests on s3utils, but with mocking we could likely enhance test coverage with some effort, but has not been done so far. When I come back to this script I will see if some tests can be added with reasonably low effort.

Reply:
Gotcha, I think no project is too small for tests but I wouldn't hold up a PR for it given my late arrival. You do you :)